service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -680,7 +680,10 @@ void wdsRecordIdNotFoundTest() throws Exception {
     assertThat(
         result.getResponse().getContentAsString(),
         containsString(
-            "Error while fetching WDS Records for Record ID(s): {MY_RECORD_ID_2=ApiException thrown for testing purposes.}"));
+            "Error while fetching WDS Records for Record ID(s): {MY_RECORD_ID_2=Message: ApiException thrown for testing purposes.\\n"
+                + "HTTP response code: 400\\n"
+                + "HTTP response body: null\\n"
+                + "HTTP response headers: null}"));
```

Comment:
the new client appears to have changed error messages. I haven't tracked this down - does this need changing?

Reply:
It's a little clunky but I don't think it's a blocker... unless the headers include things we don't want to be showing users or storing/logging

Reply:
I'm a bystander here but I would advocate making the string comparison less strict. Unless someone somewhere is dependent on exactly what the error message looks like (that is, the message is part of the API contract) I don't think you get value out of strictness here.

On the other hand, maybe we want to customize the response so it doesn't include all this extra text? Unless that's hard to do.

Reply:
I like the idea of not having the extra text in the error response message but I am not sure how hard it would be to change the response class. In the short term maybe we can update the test to be less strict?

Reply:
I will look into this a bit more, see if I can at least figure out where/why this changed.